app/assets/javascripts/angular/controllers/categories/show_category.js.erb

```diff
@@ -0,0 +1,41 @@
+app.controller('CategoryShowCtrl', ['$scope','$http','$auth','$location','$routeParams', function($scope,$http,$auth,$location,$routeParams){
+  
+  var url = "<%= ENV['URL'] %>" || "http://localhost:3000";
+
+  $http.get(url + '/categories/' + $routeParams.id).success(function(response) {
+    console.log(response);
+
+    if(response.length > 0) {
+      $scope.services = response;
+      $scope.services.forEach(function(service) {
```

Comment:
this should definitely be done in the server side and add it to `categoryController#show` as an extra parameter that it returns